### ISSUE TITLE
[.NET5] Validation of experience for Visual Basic

### ISSUE THREAD

**Issue by kvenkatrajan** [dotnet/project-system#8058]
Github mirror of [AB#1302808](https://devdiv.visualstudio.com/0bdbc590-a062-4c3f-b0f6-9383f67865ee/_workitems/edit/1302808)

- [x] #6787 (done. *)
- [ ] #6608 (done.  This is in .net core. Won't fix)
- [x] #7772
- [x] #7448  (done. *)
- [ ] #7102  (done. This is in .net core. Won't fix)
- [x] #6784
- [ ] #6659 (done. Low priority. Won't fix)
- [x] [AB#1517428](https://devdiv.visualstudio.com/0bdbc590-a062-4c3f-b0f6-9383f67865ee/_workitems/edit/1517428) ([fixed](https://devdiv.visualstudio.com/DefaultCollection/DevDiv/_git/VS/pullrequest/400783))
- [x] #3102 
- [x] #3071 
- [ ] #2706 (done. *)
- [ ] #3040 (Broken in old project system)
- [x] #2344
- [x] #8207 

\* No change needed in the old project system. It works well for .net frameworks

**Comment by ocallesp** [comment]
For the list of issues, we should only fix the ones that are in the new Project System.

cc: @kvenkatrajan

**Comment by drewnoakes** [comment]
> For the list of issues, we should only fix the ones that are in the new Project System

@ocallesp could you explain what you mean by this?

**Comment by ocallesp** [comment]
After a meeting with @melytc and @kvenkatrajan we prioritized to fix only bugs in the old project system for the old property pages. 

I confirmed with @klausloeffelmann that there are bugs in the property pages for .net framework vb

**Comment by KlausLoeffelmann** [comment]
No, that's not correct. 
There are NO bugs to my knowledge in .NET Framework, only in .NET.

**Comment by kvenkatrajan** [comment]
> After a meeting with @melytc and @kvenkatrajan we prioritized to fix only bugs in the old project system for the old property pages.
> 
> I confirmed with @KlausLoeffelmann that there are bugs in the property pages for .net framework vb

Just to confirm the expectations Osvaldo-
We would want to fix any issues identified in the legacy property pages on the legacy project system & similarly ensure that these issues cease to exist with new property pages built on top of the new project system

**Comment by KlausLoeffelmann** [comment]
So, I think the confusion here is, that the WinForms Design of the "old" Project System AppDesigner is intuitively connected to Framework Apps, while the WPF-style Pages are connected to .NET.

This is NOT true for VB!

Even for .NET, the old pages are based on the WinForms Project System AppDesigner. And _that_ scenario (only .NET!) is utterly broken. Framework (with old WinForms AppDesigner) works just fine!

**Comment by kvenkatrajan** [comment]
Yes that is correct. We are aware of that and the work to port it to the new WPF pages is being tracked and worked here in parallel with this validation effort- 
https://github.com/dotnet/project-system/issues/8057

**Comment by ocallesp** [comment]
It seems that we can close #6608 and #6787 since they are working correctly in .Net Framework for VB. (No issues in the Old Project System Old Property Pages)

**Comment by ocallesp** [comment]
From this list only #3040 and #AB1517428 should be fixed.

#AB1517428 has higher priority than #3040

cc: @kvenkatrajan @melytc

**Comment by ocallesp** [comment]
[AB#1517428](https://devdiv.visualstudio.com/0bdbc590-a062-4c3f-b0f6-9383f67865ee/_workitems/edit/1517428) fixed this [pr](https://devdiv.visualstudio.com/DefaultCollection/DevDiv/_git/VS/pullrequest/400783).
